examples_stardust/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "utils"
+version = "0.6.0"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0.62"
+identity_core = { path = "../identity_core" }
+identity_did = { path = "../identity_did" }
+identity_stardust = { path = "../identity_stardust" }
+iota-client = { version = "2.0.0-beta.2", default-features = false, features = ["tls"] }
+primitive-types = "0.11.1"
+tokio = "1.20.1"
+
+[lib]
+path = "utils/utils.rs"
+
+[[example]]
+path = "crud/0_create_did.rs"
```

Comment:
Not sure if `basic` or `crud` for the folder name is better. Unfortunately both still come after `advanced` QQ

Alternatively we could just leave the crud examples in the root of the examples folder.

Reply:
I think we'll probably have three example folders: `basic/crud`, `credentials`, `advanced`. Things like `create_vc`, `create_vp`, and `revoke_vc` can go in `credentials`, and the layer-1 interactions can stay in `advanced`.

We could always number the sub-folders to ensure correct ordering :P (unless Rust doesn't like that). `0_basic`, `1_credentials`, `2_advanced`.